config/default/core.entity_view_display.localgov_alert_banner.localgov_alert_banner.default.yml

```diff
@@ -31,25 +31,26 @@ content:
       rel: ''
       target: ''
     third_party_settings: {  }
-    type: link
+    weight: 2
     region: content
   short_description:
-    weight: 1
+    type: text_default
     label: hidden
     settings: {  }
     third_party_settings: {  }
-    type: text_default
+    weight: 1
     region: content
   title:
-    label: hidden
     type: string
-    weight: 0
-    region: content
+    label: hidden
     settings:
       link_to_entity: false
     third_party_settings: {  }
+    weight: 0
+    region: content
 hidden:
   content_moderation_control: true
+  search_api_excerpt: true
```

Comment:
@Polynya Looks like this might be a real part, but I'm struggling to see because of the noise. I wouldn't expect a change in the search config?

Reply:
This went away after pulling from develop which had some maintenance updates.